### DISCUSSION TITLE
"Input_Sea_Land_Mask" non trouvée

### DISCUSSION THREAD

**Discussion by atoulou** [orbisgis/geoclimate#929] [Q&A]
Hello,

I'm trying to use the geoprocessing tool Geoclimate for the city of Lyon (for a PhD thesis) but it doesn't work. 

I managed to run the code for Pont-de-Veyle as well as for Rennes, but for Lyon (and also for Villeurbanne) I get this error message "org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "INPUT_SEA_LAND_MASK__683B6A45_A25E_493B_A78D_BA689C51B7DF" not found". 

I understand that an input relating to maritime data is missing, but I can't find any information on the type of file that should be added (.osm, .shp.etc.), or where I should add it for resolving this issue. 

Could you please help me? 

Thanks in advance :)

**Comment by j3r3m1** [comment]
Thank your for your message and for giving the context of your GeoClimate use. If I understand well you use the default config file with "Lyon" or "Villeurbanne" instead of "Pont-de-Veyle" and you run the OSM workflow ?

**Comment by j3r3m1** [comment]
Other question, which version of the code are you using ? Just as you know, it is working well for me using the 1.0.0 version with the following configuration file:
```json
{
    "description": "Processing OSM data",
    "input": {
    	"area" : 10000,
        "locations": [
            "Villeurbanne"
        ]
    },
    "output": {
        "folder": "/tmp"
    },
    "parameters": {
        "rsu_indicators": {
            "indicatorUse": [
                "LCZ",
                "TEB",
                "UTRF"
            ],
            "svfSimplified": true,
            "estimateHeight": true
        },
        "grid_indicators": {
            "x_size": 100,
	    "y_size": 100,
	    "rowCol": false,
	    "output" : "geojson",
	    "indicators" :["BUILDING_FRACTION", "BUILDING_HEIGHT", "BUILDING_POP",
                               "BUILDING_TYPE_FRACTION", "WATER_FRACTION", "VEGETATION_FRACTION",
                               "ROAD_FRACTION", "IMPERVIOUS_FRACTION",
                               "BUILDING_HEIGHT_WEIGHTED", "BUILDING_SURFACE_DENSITY", "ASPECT_RATIO", "SVF",
                               "HEIGHT_OF_ROUGHNESS_ELEMENTS", "TERRAIN_ROUGHNESS_CLASS",
                               "LCZ_PRIMARY"]
         }
    }
}

**Reply by atoulou** [reply]
Thank you very much for your (quick!) reply. I re-run the tool with your configuration file and with version 1.0.0 of the code and it worked, so thank you. As far as I can see, the only change between your configuration file and the one I had run is the line "area:10000", could you tell me what this line is for? (for being then able to use it properly)
Thank you again for helping

**Reply by j3r3m1** [reply]
Nice to read that it works for you with the version 1.0.0.

Concerning the "area", this information can be removed, it will not change the results. By default, we limit the calculation to 1000 km². If you want to run a very large area, you might get a message telling you that your area is too big and that you need to increase this "area" to a higher value. However, having a size too big might lead to (i) fail in the downloading and (ii) very long calculations. Thus it is preferable to split a big area in several smaller (using several bounding box). But I do not think it is the case for Lyon or Villeurbanne.

**Reply by atoulou** [reply]
Thanks again for your reply. In fact, I ran the tool again without the "area line" and it worked fine, I don't know why it didn't work the first times (maybe I took the wrong version of the code) ... As for the meaning of the area line, I'm sorry but I don't understand how it works. Indeed the Lyon area for which I have outputs is 48 km², so I don't understand how the algorithm can work with the default area set to 1000m², nor how it can work with the configuration file you suggested with an area set to 10 000m², could you please explain ? 
Thanks again in advance

**Reply by j3r3m1** [reply]
Sorry for the mistake, I have updated my comment. It was 1'000 km²...

**Reply by atoulou** [reply]
Perfect, I understand everything now, thank you very much!
